package.json

```diff
@@ -28,6 +28,10 @@
         "test": "npm run test -w packages/ --if-present",
         "testWeb": "npm run testWeb -w packages/ --if-present",
         "testE2E": "npm run testE2E -w packages/ --if-present",
+        "test:ui:prepare": "./node_modules/.bin/extest get-vscode -s ~/.vscode-test-resources -n && extest get-chromedriver -s ~/.vscode-test-resources -n",
```

Comment:
its not entirely clear to me when each of these commands should be run and how they are distinguished. Do you mind updating the PR description to give some more context here? 

Ex. What order are they run? Why are they split up? etc. 

I suspect they are very good reasons for this, but looking at the commands its not obvious.